pkg/transfer/local/pull.go

```diff
@@ -255,24 +255,22 @@ func (ts *localTransferService) pull(ctx context.Context, ir transfer.ImageFetch
 }
 
 func fetchHandler(ingester content.Ingester, fetcher remotes.Fetcher, pt *ProgressTracker) images.HandlerFunc {
-	return func(ctx context.Context, desc ocispec.Descriptor) (subdescs []ocispec.Descriptor, err error) {
+	return func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
```

Comment:
This function appears to have been copied more-or-less wholesale from `FetchHandler` in `handlers`; my thoughts at https://github.com/containerd/containerd/pull/9155/files#r1339001829 apply equally here.